src/allmydata/test/cli_node_api.py

```diff
@@ -134,7 +134,7 @@ class CLINodeAPI(object):
 
     @property
     def twistd_pid_file(self):
-        return self.basedir.child(u"twistd.pid")
+        return self.basedir.child(u"running.process")
```

Comment:
We can make all kinds of improvements to the code.

In such .. "legacy" .. code I tend to try _not_ to refactor "too much" -- because there's also pressure in tahoe to not have "big" PRs.

I did try some "small" refactors for this and the other one -- but since FilePath usage isn't consistent, that means also changing yet more things (I didn't explore how many). So, no, not doing these right now :/